api.js

```diff
@@ -122,7 +124,13 @@ var v2 = {
             emitter = new events.EventEmitter();
         }
 
-        var req = http.request(options, function(res) {
+        let urlinfo = url.parse(config.standup.url);
+        options.host = urlinfo.hostname;
+        options.port = urlinfo.port;
+        options.protocol = urlinfo.protocol;
+        var request = (options.protocol === 'https:' ? https : http).request;
```

Comment:
Because lazy and I realized half way through that I could use es6. Need to
do a larger es6 sweep actually. Let is just better in a lot of ways.